### PR TITLE
fix(ui): pagination active page styles

### LINKED ISSUE #198: Fix pagination styles
**Problem**
On grid pages:
- we add `disabled` class for currently opened page, unlike in Tabler
- there is border at the right end, unlike in Tabler

CleanShot 15 11 2024 at 15-18

[Tabler](https://preview.tabler.io/pagination.html#):

CleanShot 15 11 2024 at 15-19

**Solution**
- remove border at the right end
- add styles for active link as in Tabler, but keep it disabled so that user will not be able to reload page, focus it via tab, etc.

### PR DESCRIPTION
**Pull request by ksnitsky**
Closes #198.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/views/ats/candidates/activities_tab.html.slim [hunk 1]
```diff
@@ -4,4 +4,4 @@
       .divide-y
         - @all_activities.each do |event|
           = candidate_display_activity(event)
-  = paginate(@all_activities, theme: "ats", klass: "mt-3")
+  = paginate(@all_activities, theme: "ats")
```
Comment (ksnitsky) [inline app/views/ats/candidates/activities_tab.html.slim:7]: https://github.com/freeats/freeats/pull/208#discussion_r1846051356

File: app/views/kaminari/ats/_paginator.html.slim [hunk 2]
```diff
@@ -7,7 +7,7 @@
     paginator:     the paginator that renders the pagination tags inside
     class:         additional classes for wrapper
 == paginator.render do
-  nav.page-paginator.mt-3 aria-label="pager" class=local_assigns[:class]
+  nav.mt-3 aria-label="pager" class=local_assigns[:class]
```
Comment (ksnitsky) [inline app/views/kaminari/ats/_paginator.html.slim:10]: `page-paginator` without styles removed from `datagrid.scss` is useless

**Comment by ksnitsky** [comment]
In #208, the top paginator on the candidate grid is misaligned
image

File: app/assets/stylesheets/src/shared/_datagrid.scss [hunk 3]
```diff
@@ -74,3 +29,9 @@ a.table-sort {
   min-width: 140px;
   width: 140px;
 }
+
+.page-paginator {
+  &:not(&:first-child) {
+    margin-top: 16px;
+  }
+}
```
Comment (ksnitsky) [inline app/assets/stylesheets/src/shared/_datagrid.scss:37]: I looked at the markup of all the grids and the top pagination is always the first child of the parent element, and the bottom pagination is always not the first (always after the table)

**Review by Lyams — APPROVED** [review]

**Pull request merged**